pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1756,6 +1756,110 @@
       heading: "Don't see your country?"
       desc: "While we currently do not actively recruit new partners into our professional learning program, we encourage organizations worldwide to join us in organizing the annual Hour of Code campaign as the first step to introducing computer science in your country. [Learn more about a partnership opportunity and apply here!](%{apply_url})"
 
+  tools_pages_shared:
```

Comment:
These strings will be used in views that can be shared across the three new tools pages.

Reply:
This may not be worth it, but I'm wondering if these could be combined? For example:

```
tools_pages:
  shared:
    glance:
      titles:
        ages: "Ages"
        level: "Level"
        make: "What you can make"
        devices: "Devices"
        browsers: "Browsers"
        languages: "Languages supported"
        accessiblity: "Accessibility"
      button: "View documentation"
    featured_projects:
      heading: "Featured student projects"
      desc: "Explore more featured projects built by students like you in our featured project gallery."
  applab_page:
    heading: "App Lab"
    subheading: "Create an app with Javascript"
```

It's only worth it if it would help the international team group all of the translations for these pages though.

Reply:
Oh interesting! My only hesitation is that I think it'll make all of these strings extra long in the markup. 

@mgc1194 do you have any thoughts on whether this pattern would be beneficial?

Reply:
I thought about this more, and I'm going to keep what I had originally. There will be three more similar pages added, and I worry it'll get too nesty and confusing!